index.html

```diff
@@ -3,9 +3,94 @@
   <head>
     <link rel="stylesheet" href="app.css"
     <meta charset="utf-8">
-    <title></title>
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.4.0/css/font-awesome.min.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/normalize/3.0.3/normalize.css">
+    <title>Surf & Paddle Co</title>
   </head>
   <body>
+<div class="container">
+  <ul class="left-top-logo">
+<li class="fa fa-pie-chart"></li>
+<li class="surf">SURF & PADDLE CO. BLOG</li>
+</ul>
+</nav>
+    <ul class="right-top-nav">
+     <li>ABOUT</li>
+      <li>SEARCH</li>
+      <li><i class="fa fa-search"></i></li>
+    </ul>
+  </nav>
+  <div class="header">
+    <h1 class="main-title">SURF & PADDLE CO.</h1>
+    <p class="title-sub">An Ironyard Academy Project Est. Fall 2013</p>
+  </div>
+</div>
+<section class="left-section">
+   <h1 class="tit1">A Fantastic Engaging Storyline Goes Here</h1>
+   <img class="icon-section" src="http://placehold.it/25/25" alt="Icon"/>
+   <p class="name1">Chancey Ward</p>
 
-  </body>
+   <p class="underline">on <span>Monday Oct 7th 9:22pm</span></p>
+   <p class="content">Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.</p>
+   <p class="content">Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.</p>
+   <h3 class="tit2"> WOAH, THAT'S RADICAL!</h3>
```

Comment:
can you give a example of a block quote I really am sorry I know you want a weekend. I'm just trying my best to get good at this. and as far as resizing the main pics on the bottom do I grab them all as a group and set width and height and should they be inline?

Sent from Yahoo Mail on Android

From:"Ryan Tablada" <notifications@github.com>
Date:Sat, Oct 24, 2015 at 10:39 AM
Subject:Re: [05-surf-and-paddle] Turning in 05-surf-and-paddle (#1)

In index.html:

> > - </body> > + <p class="underline">on <span>Monday Oct 7th 9:22pm</span></p> > + <p class="content">Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.</p> > + <p class="content">Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt
 in culpa qui officia deserunt mollit anim id est laborum.</p> > + <h3 class="tit2"> WOAH, THAT'S RADICAL!</h3> 

This is probably better as a blockquote element

—
Reply to this email directly or view it on GitHub.￼